romana-install/roles/cache/apply/tasks/repo_ubuntu_trusty.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Synchronize apt cache to hosts
+  synchronize: src="{{ cache_dir }}/{{ ansible_distribution|lower }}_{{ ansible_distribution_release|lower }}_apt/" dest="/var/cache/apt/archives/"
```

Comment:
The expression "{{ ansible_distribution|lower }}_{{ ansible_distribution_release|lower }}" appears multiple times in different places. Could this just be stored in a single, short variable once and then this variable is used instead of the complex expression?